### PR TITLE
Add quicksilver assets to Osmosis f/e

### PR DESCRIPTION
**Pull request by joe-bowman**
Add tokens qjuno, qsaga, qdydx, qbld from Quicksilver to osmosis.zone_assets.json

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -3914,6 +3917,42 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-05-08T18:00:00Z",
       "_comment": "$WEIRD Friends NFTs"
+    },
+    {
+      "chain_name": "quicksilver",
+      "base_denom": "uqjuno",
+      "path": "transfer/channel-522/uqjuno",
+      "osmosis_verified": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3925]: Please always set to false upon initial listing. We can update to true after, but we have to validate everything first.

**Comment by JeremyParish69** [comment]
Adding Assets
If adding a new asset, please ensure the following:

[x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
[x] Add asset to bottom of zone_assets.json.
[x] chain_name and base_denom are provided and use values exactly as defined at the Chain Registry.
[x]  path is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
[ ] osmosis_verified is set to false
[ ] listing_date_time_utc is specified and accurate
[ ] Optional: transfer_methods, peg_mechanism, override_properties, canonical, categories, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).

**Comment by joe-bowman** [comment]
What does the listing date refer to? I had copied an existing record and it
was missing these fields.

*Joe Bowman*
Founder, Quicksilver Protocol
***@***.***

**Comment by JeremyParish69** [comment]
> What does the listing date refer to? I had copied an existing record and it was missing these fields.

It refers to the date the asset was listed on Osmosis Zone, and is used to show which assets are new and assist with tracking going forward. We don't have a full backlog, which is why some older assets, inlcuding, qregen may be missing its listing date.

**Comment by JeremyParish69** [comment]
Please see the [requirements](https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md#upgrade-asset-to-verified-status-permissioned) for Upgrading to Verified Status, and another PR can implement the update.

**Pull request merged**